deploy/helms/airflow/templates/worker-rbac.yaml

```diff
@@ -0,0 +1,29 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: "{{ .Release.Name }}-airflow-worker"
+rules:
+- apiGroups: [""] # core API group
+  resources: ["configmaps", "secrets"]
```

Comment:
I guess worth limit access only to airflow specific resources with resourceNames filter

Reply:
Airflow worker should be able to read configmaps to get DAGs configuration.